applications/asset_tracker_v2/src/cloud/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-target_include_directories(app PRIVATE .)
+zephyr_include_directories(app PRIVATE .)
```

Comment:
Seems an earlier comment here somehow was lost.

Why is this suddenly targeting `zephyr_interface` library (and thus a global include folder), and not `app` lib ?

Secondly, the use of `app PRIVATE` is wrong when using `zephyr_include_directories()` or `zephyr_library_include_directories()`.

Reply:
Turns out, this change is not necessary with the recent changes. But, I needed to use
`zephyr_include_directories_ifdef` and `zephyr_include_directories` in the `CMakeLists.txt` files of the various sample/apps to properly import each `x-cert.pem` file. If I used the `target_` flavour the files were never found by the cloud backend. Is it ok this way? Or do you suggest an alternative path?